github/libp2p/membership.json

```diff
@@ -176,42 +104,21 @@
   "jbenet": {
     "role": "admin"
   },
-  "jbenetsafer": {
```

Comment:
What is the purpose of this account? Anyone know?

Reply:
@jbenet @jbenetsafer does this account still require access to the libp2p org?

Reply:
Ping him on slack, he doesn't read github notifications.

Reply:
The account is not being removed from the org anymore - just moved to the alumni team.